salt/metalk8s/repo/offline.sls

```diff
@@ -59,6 +63,7 @@ Refresh yum cache:
 Repositories configured:
   test.succeed_without_changes:
     - require:
+      - file: Set osmajorrelease in yum vars
```

Comment:
For me it's the `Configure {{ repo_name }} repository` that should require this.

(Also for me you should use `require_in` in the for loop instead of doing 2 for loop

Reply:
Yeah you're right for the first one and for the `require_in` I wanted to do it and totally forgot.
I will change this.